features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/messagecomposer/MessageComposerView.kt

```diff
@@ -71,10 +71,14 @@ internal fun MessageComposerView(
         }
     }
 
-    fun onVoiceRecordButtonEvent(press: PressEvent) {
+    val onVoiceRecordButtonEvent = { press: PressEvent ->
```

Comment:
Actually here it shouldn't make a difference, but generally I'm starting to prefer `val` over `fun` so that changes in captured variables will trigger recomposition. With `fun` I've found that subtle bugs can sneak in easily.

Reply:
https://gist.github.com/jonnyandrew/fd8d2762e2f6fcdb1656ea35c6da5eb9